deployments/config/galactic-sovereign-service.yml

```diff
@@ -6,17 +6,17 @@ http:
       rule: 'Host(`api.{{ env "DOMAIN_NAME" }}`) && PathPrefix(`/v1/galactic-sovereign`)'
       service: galactic-sovereign-service
       middlewares:
-        - "rate-limit"
```

Comment:
It's unclear if the middlewares' names need to be unique or not. But during local testing when having multiple `rate-limit` defined (for this service and for the `user-service` we saw this log message from `traefik`:

![image](https://github.com/user-attachments/assets/fab74751-a318-4a34-a852-c1aada9161bd)

This also persists when the configuration of both middlewares is not exactly the same. This might indicate that the names have to be unique. The documentation around [middlewares](https://doc.traefik.io/traefik/middlewares/overview/) is not very explicit.

As a precautionary measure we modified the names of the middlewares to be unique.